css-four-column-layout/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Four Column Layout</title>
+  <link rel="stylesheet" href="styles.css">
+  <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <div class="row justify-center">
+      <h2 class="font font-weight-heading font-size-heading ">How It Works</h2>
+    </div>
+    <div class="row">
+      <img src="images/screen-1.png" class="justify-center image-sizing">
+      <img src="images/screen-2.png" class="justify-center image-sizing">
+      <img src="images/screen-3.png" class="justify-center image-sizing">
+      <img src="images/screen-4.png" class="justify-center image-sizing">
+    </div>
+    <div class="row justify-space-around">
+      <img src="images/switch-icon.png" class="small-image-sizing">
+      <img src="images/filter-icon.png" class="small-image-sizing">
+      <img src="images/slider-icon.png" class="small-image-sizing">
+      <img src="images/bars-icon.png" class="small-image-sizing">
+    </div>
+    <div class="row justify-space-around">
+      <h3 class="font font-weight-heading">App Switching</h3>
+      <h3 class="font font-weight-heading">Trip Filtering</h3>
+      <h3 class="font font-weight-heading">Service Filters</h3>
+      <h3 class="font font-weight-heading">Stats + Milage</h3>
+    </div>
+    <div class="row justify-space-around">
+      <p class="font">Toggles all your apps on and off.</p>
+      <p class="font">Evaluates your trip requests.</p>
+      <p class="font">Accepts only your ideal trips.</p>
+      <p class="font">Tracks your trips across all apps.</p>
+    </div>
```

Comment:
I can see why you would do this as related to layout classes, but organized like this, it seems like the content of each of these rows is not related data, when in fact they are all related. You will want one row with four columns, each of which holding the phone image, icon, and text within it.